Common/Operations/Nightscout.swift

```diff
@@ -329,7 +328,7 @@ public extension Site {
                 updatedSite.deviceStatuses = deviceStatus
             }
             
-            updatedSite.lastUpdatedDate = Date()
+            updatedSite.nextRefreshDate = Date()
```

Comment:
Does it make sense to still call this "nextRefreshDate"? We're putting the date it was last updated here. Also... should we calculate the nextRefreshDate from the last SVG record we received?